src/DynamoCoreWpf/ViewModels/Watch3D/DynamoGeometryModel3D.cs

```diff
@@ -289,6 +289,10 @@ private DynamoMeshVertex[] CreateVertexArray()
             var vertexCount = geometry.Positions.Count;
             var result = new DynamoMeshVertex[vertexCount];
 
+            var isSelected = (bool)GetValue(AttachedProperties.ShowSelectedProperty);
+            var selectiveMode = (bool)GetValue(AttachedProperties.SelectivePreviewProperty);
+            var isSpecialPackage = (bool)GetValue(AttachedProperties.IsSpecialRenderPackageProperty);
+
```

Comment:
Few notes for the above...

1. To align the name with this feature, should `selectiveMode` be `isolationMode`? That may avoid the confusion due to another `isSelected` variable above.
2. Not sure what `isSpecialPackage` mean, can we rename it to `inheritsTransparencyValue` or something like that? All we can do as readers here is to guess its meaning.